ldsc/src/main/resources/translatePartitionedHeritability.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+import glob
+import json
+import math
+import numpy as np
+import os
+import re
+from scipy.stats import t as tdist
+import subprocess
+
+downloaded_files = '/mnt/var/ldsc'
+s3_path = 's3://dig-analysis-data/out/ldsc/partitioned_heritability'
+
+
+def get_phenotype_annot_ancestries():
+    out = {}
+    for file in glob.glob(f'{downloaded_files}/*/*/*'):
+        ancestry, phenotype, annot = re.findall('.*/(.+).results', file)[0].split('.')
+        if phenotype not in out:
+            out[phenotype] = {}
+        if annot not in out[phenotype]:
+            out[phenotype][annot] = []
+        out[phenotype][annot].append(ancestry)
+    return out
+
+
+def translate(file):
+    with open(file, 'r') as f:
+        # Get last line of file
+        last_line = [a.strip() for a in f.readlines()[-1].strip().split('\t')]
```

Comment:
Not sure how big this files are, but rather that reading all the lines in the file into a list and then taking the last one you could use the tailer library:

```
last_line = [a.strip() for a in tailer.tail(f, num_lines=1)][0].split('\t')]
```

Reply:
My understanding is it won't incur the cost of reading the whole file into memory.

Reply:
These files are set at about 100 lines. Typically elsewhere I do dynamically read line by line with f.readline() and then bounce when the line size is zero. Seemed like overkill here.

Reply:
Oh I see, yeah could. I don't really want to use a library just for this though.

Reply:
Yeah, I get that.  Files seems small enough to leave things as they are.